src/test/java/racingcarTest/util/StringValidatorTest.java

```diff
@@ -0,0 +1,54 @@
+package racingcarTest.util;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import racingcar.util.StringValidator;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class StringValidatorTest {
```

Comment:
`@DisplayName`에 "validateIsEmpty 메소드에" 라는 내용이 겹치는데, 이런 식으로 클래스에 통째로 묶어내는 방법은 어떨까요?? 😀
```suggestion
@DisplayName("validateIsEmpty 메소드에")
public class StringValidatorTest {
```

Reply:
선권님 코드리뷰보며 배웠는데, 한번더 집어 주셨네요!!! 넵 알겠습니다~~~